components/automate-gateway/gateway/middleware/auth_interceptor_internal_test.go

```diff
@@ -74,43 +74,6 @@ func TestHeaderAuthValidForClientAndPeer(t *testing.T) {
 	}
 }
 
-func TestGetProjectsFromMetadata(t *testing.T) {
```

Comment:
Heh I wasn't aware we had these tests. They're a bit different from the ones I've created when refactoring this helper: https://github.com/chef/automate/blob/c435118f939f11e1ed09a30d6f4edab24b6b1aa4/lib/grpc/auth_context/auth_context_test.go#L139
😃 It's fine though.